.kuzzlerc

```diff
@@ -95,20 +134,25 @@
         }
       }
     },
-    "guest": {
-      "_id": "guest",
+    "anonymous": {
+      "_id": "anonymous",
       "indexes": {
-        "_canCreate": true,
+        "_canCreate": false,
         "*": {
           "_canDelete": true,
           "collections": {
-            "_canCreate": true,
-            "_canDelete": true,
+            "_canCreate": false,
+            "_canDelete": false,
```

Comment:
should be moved under the wildcard:

    "collections": {
      "_canCreate": false,
      "*": {
        "_canDelete": false,
        (...)
      }
    }